js/controllers/grid/navigationMenus/form/NavigationMenuItemsFormHandler.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * @file js/controllers/grid/navigationMenus/form/NavigationMenuItemsFormHandler.js
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class NavigationMenuItemsFormHandler
+ * @ingroup js_controllers_grid_navigationMenus_form
+ *
+ * @brief NavigationMenuItems form handler.
+ */
+(function($) {
+
+
+	/**
+	 * @constructor
+	 *
+	 * @extends $.pkp.controllers.form.AjaxFormHandler
+	 *
+	 * @param {jQueryObject} $formElement A wrapped HTML element that
+	 *  represents the approved proof form interface element.
```

Comment:
This variable description is incorrect.

Reply:
ok

Reply:
This doc line still needs to be updated.

Reply:
OK